recipes/stemflow/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "stemflow" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 194af6f1247875eb7e0eadbec327f29e29d569484337cc45e003b7a565315814
+
+build:
+  number: 0
+  skip: true  # [win, osx]
```

Comment:
why are you skipping win and osx? If that's not necessary, then this should probably be `noarch: python`:
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
Thanks for the hint. Fixed.